Source/WebCore/Modules/identity/IdentityRequestProvider.h

```diff
@@ -26,11 +26,13 @@
 #pragma once
 
 #include "IdentityCredentialProtocol.h"
-
+#include <JavaScriptCore/JSObject.h>
+#include <JavaScriptCore/Strong.h>
```

Comment:
Another blank line here
```suggestion
#include <JavaScriptCore/Strong.h>

```